### PR TITLE
Remove -contrib features and install packages in onCreateCommand.sh

### LINKED ISSUE #7412: Consider removing dependencies on -contrib Codespace features
We currently depend on the two -contrib codespace features, which are maintained by a developer in private repos:
- ghcr.io/devcontainers-contrib/features/ant-sdkman:2
- ghcr.io/devcontainers-contrib/features/tmux-apt-get:1

The developer maintaining each contrib feature is listed here:
https://containers.dev/features

But I can't find a public repo of the actual container, just links to the container layer that's added to our dev container.

It might be good to install these dependencies ourselves instead of relying on a container built in a private repo.

### PR DESCRIPTION
**Pull request by avaleske**
### Description

- Remove -contrib codespace features that I couldn't find a public repository for
  - tmux we install manually
  - the sdkman feature was redudant since we're already installing java with ghcr.io/devcontainers/features/java:1.4.1
- Manually installs tmux and moves other package installation to `onCreateCommand.sh`

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)
- [x] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [x] Extended the README / documentation, if necessary

### Instructions for manual testing

If instructions are needed for manual testing by reviewers, include them here.

### Issue(s) this completes

Fixes #7412

### CODE REVIEW

**Review by gwendolyngoetz — APPROVED** [review]

**Comment by avaleske** [comment]
@dkatzz looks like it auto merged after gwen approved, lmk if there's any changes you want and I'll make them in a followup.

**Comment by dkatzz** [comment]
> @dkatzz looks like it auto merged after gwen approved, lmk if there's any changes you want and I'll make them in a followup.

LGTM thank you!

**Pull request merged**
